thoth/lab/adviser.py

```diff
@@ -339,7 +343,9 @@ def create_adviser_heatmap(
             os.makedirs(project_dir_path, exist_ok=True)
 
             fig = ax.get_figure()
-            fig.savefig(f"{project_dir_path}/Adviser_justifications_{datetime.now()}.png", bbox_inches="tight")
+            fig.savefig(
+                f"{project_dir_path}/Adviser_justifications_{file_name}_{datetime.now()}.png", bbox_inches="tight"
```

Comment:
having "" as default seems to be not very good at this point. Also, datatime.now() generated timezone and locale specific results, isn't it? maybe setting the locale and TZ would be good.